cli/formatter/table.go

```diff
@@ -24,15 +24,41 @@ func decodeYamlArr(input string) ([]any, error) {
 	return decoded, nil
 }
 
-// castAnyMapToStringMap casts map[any]any to map[string]any.
-func castAnyMapToStringMap(src map[any]any) map[string]any {
-	dst := make(map[string]any)
+// castAnyMapToStringUMap casts map[any]any to unorderedMap[string].
+// The result will be sorted by keys.
+func castAnyMapToStringUMap(src map[any]any) unorderedMap[string] {
```

Comment:
Sorry, but I don't get the idea: why do we need unorderedMap[string] and unorderedMap[any] at the same time? Why do not enough unorderedMap[any]?

Reply:
Removed all unorderedMap[string] (unorderedMap[any] is used).